realm/src/main/java/io/realm/RealmResults.java

```diff
@@ -490,6 +491,18 @@ public boolean add(E element) {
     public void add(int index, E element) {
         throw new UnsupportedOperationException();
     }
+
+    /**
+     * Returns this result set as a {@link RealmCursor}. All changes done to this RealmResults afterwards will
```

Comment:
"Returns the RealmResults as a RealmCursor. All changes applied to this RealmResults ..."